.openapi-generator/FILES

```diff
@@ -5,6 +5,7 @@ equinix/services/fabricv4/api/__init__.py
 equinix/services/fabricv4/api/cloud_routers_api.py
```

Comment:
I think we exclude this file in equinix-sdk-go; should do the same in the Python SDK.